lambdaclass/mirra_backend
# Gateway tests

Implement some basic integration tests. Tests should be expanded later but this serves as a sort of template for them in the future.

# To test

Run `mix test`. No further setup should be needed. Let me know if that does not work on your machine!
